x-pack/plugins/security_solution/public/common/containers/sourcerer/index.test.tsx

```diff
@@ -188,6 +190,8 @@ describe('Sourcerer Hooks', () => {
           type: 'x-pack/security_solution/local/sourcerer/SET_SOURCERER_SCOPE_LOADING',
           payload: { loading: false },
         });
+        expect(mockDispatch).toHaveBeenCalledTimes(7);
```

Comment:
this was getting called 12 times before because of the bug [fixed here](https://github.com/elastic/kibana/pull/120814/files#r767993949)